### PR TITLE
Fix Lambda inline editor not showing up

### PR DESCRIPTION
**Pull request by parostatkiem**
**Description**

Changes proposed in this pull request:

- force redraw the editor on visibility change
- ...
- ...

**Related issue(s)**

### CODE REVIEW

**Review by akucharska — DISMISSED** [review]

**Review by akucharska — DISMISSED** [review]

File: core-ui/src/components/Lambdas/LambdaDetails/Tabs/Code/CodeAndDependencies/Editor.js [hunk 1]
```diff
@@ -27,7 +40,17 @@ export default function Editor({
     };
   }, []);
 
+  useEffect(() => {
+    if (observer) observer.observe(editorContainer.current);
+    // add intersection observer to both versions of the editor
+    else
+      console.warn(
+        'Could not apply IntersectionObserver to code editor. Visibility problems may occur.',
+      ); // eslint-disable-next-line react-hooks/exhaustive-deps
```
Comment (aerfio) [inline core-ui/src/components/Lambdas/LambdaDetails/Tabs/Code/CodeAndDependencies/Editor.js:49]: add `{` and `}` to that if and else

File: core-ui/src/components/Lambdas/LambdaDetails/Tabs/Code/CodeAndDependencies/Editor.js [hunk 2]
```diff
@@ -27,7 +40,17 @@ export default function Editor({
     };
   }, []);
 
+  useEffect(() => {
+    if (observer) observer.observe(editorContainer.current);
+    // add intersection observer to both versions of the editor
+    else
+      console.warn(
```
Comment (aerfio) [inline core-ui/src/components/Lambdas/LambdaDetails/Tabs/Code/CodeAndDependencies/Editor.js:47]: change that to error please, there's too many warnings that we ignore, and such a thing is important enough to be error

**Review by aerfio — CHANGES_REQUESTED** [review]
LGTM, just 2 small changes needed

File: core-ui/src/components/Lambdas/LambdaDetails/Tabs/Code/CodeAndDependencies/Editor.js (hunk 2 above)
Comment (parostatkiem) [reply core-ui/src/components/Lambdas/LambdaDetails/Tabs/Code/CodeAndDependencies/Editor.js:47]: I used warning intentionally. In my opinion it is definitely not an error. Whoever uses a stone-age browser has to be ready for an inconvenience 🤣

**Review by aerfio — APPROVED** [review]

**Pull request merged**